hindi/Sunny_Deol/sunny.md~

```diff
@@ -0,0 +1,5 @@
+Sunny Deol is eldest son of [Dharmendra](https://en.wikipedia.org/wiki/Dharmendra).
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.